integrationtest/backend-tck/src/main/java/org/hibernate/search/v6poc/integrationtest/backend/tck/ObjectFieldStorageIT.java

```diff
@@ -309,7 +305,6 @@ private void initData() {
 			// This object will trigger the match; others should not
 			object = accessors.self.add( document );
 			accessors.string.write( object, MATCHING_STRING );
-			accessors.string.write( object, NON_MATCHING_STRING );
```

Comment:
Well, I removed this line because it's useless and looks like a mistake.

We are not testing multi-values for fields so I don't see the point of it.

Reply:
Right, we don't test multi-valued fields in other tests.

However, nested fields have everything to do with multi-valued fields... they are only useful when a given sub-fields has multiple values, otherwise a flattened model would work just as well.

So  in my opinion there is value in testing multi-valued fields here: there is bound to be some special handling of multi-values in the backend (maybe not our code, but somewhere), so I'd rather take that opportunity to confuse the backend.

If you really think it's better without multi-valued fields... we also have multi-valued fields below, so if we don't want multi-valued fields in a given object, we might as well clean up everything, not just this particular occurrence.

Reply:
Hu? Nested objects make sense when there are multiple **objects** for a given property.

I don't see why they would make sense for a multi-valued string field.

Anyway, my point is that this line seems to be useless for the tests we have. At least, I don't see how it is related to them